generic3g/specs/FieldSpec.F90

```diff
@@ -181,17 +181,15 @@ function new_FieldSpec_varspec(variable_spec) result(field_spec)
 
    end function new_FieldSpec_varspec
 
-   subroutine set_geometry(this, geom, vertical_grid, timestep, rc)
+   subroutine set_geometry(this, geom, vertical_grid, rc)
       class(FieldSpec), intent(inout) :: this
       type(ESMF_Geom), optional, intent(in) :: geom
       class(VerticalGrid), optional, intent(in) :: vertical_grid
-      type(ESMF_TimeInterval), optional, intent(in) :: timestep
       integer, optional, intent(out) :: rc
 
       integer :: status
 
       call target_set_geom(this, geom, vertical_grid)
-      call target_set_timestep(this, timestep)
```

Comment:
You can delete the subroutine `target_set_timestep` unless it is used elsewhere.